tubes/queue.py

```diff
@@ -0,0 +1,135 @@
+# -*- test-case-name: tubes.test.test_queue -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+The QueueFount.
+"""
+
+from collections import deque
+
+from zope.interface import implementer
+from twisted.internet import defer, task
+from twisted.python.failure import Failure
+
+from .kit import Pauser, beginFlowingTo
+from .itube import IFount, StopFlowCalled
+
+
+class NotABigTruckError(Exception):
+    """
+    A series of tubes is not just a big truck you
+    can dump stuff onto...
+    """
+    pass
+
+
+
+@implementer(IFount)
+class QueueFount(object):
+    """
```

Comment:
The whole point of the `series` API is that you can be totally ignorant of queuing theory and still write stuff that respects backpressure :)